### ISSUE TITLE
Comic-DL not downloading certain pages.

### ISSUE THREAD

**Issue by Aemilian666** [Xonshiz/comic-dl#267]
Trying to download:

https://www.comicextra.com/comic/ultimate-comics-spider-man-2011 

And it stops right before 16.5

Why are we not able to download any issues with periods in the number, and why does it seem to block every download thereafter from downloading? This happened with Ultimate SPider Man 200 as well, and since Issue 0.5 was the first one on that page, I couldn't download a damn thing.

I fixed  this issue by changing the order from Descending to Ascending, but this won't help for long, especially when it gets to #16 because #16.5 is the next one.

Whoops! Seems like I can't connect to website.
It's showing : <Response [404]>
Run this script with the --verbose argument and report the issue along with log file on Github.
ERROR:root:Error downloading : https://www.comicextra.com/ultimate-comics-spider-man-2011/chapter-16.1/full
[Comic-dl] Auto processing:  50%|########        | 1/2 [01:31<01:31, 91.61s/comic, comic_name=https://www.comicextra.com/comic/ultimate-spider-man-2000][Comic-dl] Auto processing with error for https://www.comicextra.com/comic/ultimate-spider-man-2000 : list index out of range
[Comic-dl] Auto processing: 100%|################| 2/2 [01:31<00:00, 64.23s/comic, comic_name=https://www.comicextra.com/comic/ultimate-spider-man-2000]

So basically, we cannot download single issues. Ultimate Spider Man was not found as a Search Comic. I had to use the actual URL.

When do you plan to implement ranges with decimals?

**Comment by Aemilian666** [comment]
Unnecessary issue.
